src/timeatlas/time_series/time_series.py

```diff
@@ -201,6 +202,18 @@ def prepend(self, ts: 'TimeSeries'):
         new_series = concat([ts.series, self.series])
         return TimeSeries(new_series, self.metadata)
 
+    def chunkify(self, n: int) -> List['TimeSeries']:
+        """
+        Splits TimeSeries into chunks of length n
+
+        :param n: length of chunks
+        :return: List of TimeSeries
```

Comment:
Please make Google Docstrings as stated in: https://timeatlas.dev/contributing/coding_style_guide.html

It's possible to set that up in your IDE (IntelliJ or PyCharm) in Preferences > Tools > Python Integrated Tools > Docstrings > Docstrings format, then choose Google.